src/components/RoadmapList/Select.tsx

```diff
@@ -0,0 +1,55 @@
+import Dropdown, { Option } from 'react-dropdown';
+
+import Icon from 'src/components/ui/Icon';
+
+import 'react-dropdown/style.css';
+import style from './Select.module.css';
+
+interface SelectProps {
+  placeholder: string;
+  options: string[];
+  onSelect: (name: string, value: string) => void;
+  iconId: string;
+  name: string;
+  value: string;
+}
+
+const Select = ({
+  value,
+  placeholder = 'Category',
+  options,
+  onSelect,
+  iconId,
+  name,
+}: SelectProps) => {
+  const handleSelect = (data: Option) => {
+    onSelect(name, data.value);
+  };
+
+  return (
```

Comment:
we need add click outside in this select

Reply:
This select is only used for filters, the handleSelect function has been added to ease changing values outside the component. Or I did not understand what this comment means

Reply:
I mean if user open select and click to to other element on page select don't close. And ask about default value, because a can't reset one filter, only all when click reset button